odoo/addons/base/ir/ir_translation.py

```diff
@@ -150,7 +152,9 @@ def finish(self):
                                src = ti.src,
                                state = 'translated'
                            FROM %s AS ti
-                           WHERE %s AND ti.value IS NOT NULL AND ti.value != ''
+                           WHERE %s AND ti.value IS NOT NULL
+                           AND ti.value != ''
+                           AND noupdate IS false
```

Comment:
Just a matter of style, I suggest to split conditions as one line per column (see below).
About `IS FALSE`, I checked Postgresql's documentation: the operator `IS` with booleans "always returns true or false, never a null value, even when the operand is null." Therefore, you can select the cases where `noupdate` is either false or null as `noupdate IS NOT TRUE`.
```sql
    WHERE %s
    AND ti.value IS NOT NULL AND ti.value != ''
    AND noupdate IS NOT TRUE
```